lib/codebreaker/validations/validations.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Codebreaker
+  module Validations
+    MAX_LENGTH = 20
+    MIN_LENGTH = 3
+
+    def self.included(base)
```

Comment:
Imho u don't need it here, just include or extend your module

Reply:
fixed